fiftyone/operators/delegated.py

```diff
@@ -301,3 +301,5 @@ async def _execute_operator(self, doc):
 
             if not is_generator:
                 return raw_result
+            else:
+                all(raw_result)
```

Comment:
I'm not familiar with this code but is it safe to assume that the generator always returns something truthy? `all()` would short circuit if it encounters something falsey.

Just use this for safety?

```py
for _ in raw_result:
    pass
```